contracts/world/sources/character/character.move

```diff
@@ -61,27 +73,34 @@ public fun create_character(
     registry: &mut CharacterRegistry,
     _: &AdminCap,
     game_character_id: u32,
+    tenant: String,
     tribe_id: u32,
     name: String,
     _: &mut TxContext,
 ): Character {
```

Comment:
following my previous comment (and it's actually more annoying for me) but maybe it's cleaner to pass the struct there?


```move
public fun create_character(
  registry: &mut CharacterRegistry,
  _: &AdminCap,
  game_character_id: GameId,
  tribe_id: GameId,
  name: String,
  _: &mut TxContext,
): Character {
```